service/src/main/java/uk/gov/hmcts/reform/fpl/config/email/MtaMailSenderConfiguration.java

```diff
@@ -18,13 +18,13 @@
 @Configuration
 @ConfigurationProperties(prefix = "spring.mail")
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
-public class EmailConfiguration {
+public class MtaMailSenderConfiguration {
     private String host;
     private int port;
 
     private final SmtpPropertiesConfiguration smtpPropertiesConfiguration;
 
-    @Bean
+    @Bean("mtaMailSender")
```

Comment:
minor: you can just rename the signature of the method instead of annotating

Reply:
i think that annotation is basically safer in such instance, in case somebody decide to refactor method name

Reply:
good point. maybe to align the name of the method with the explicit bean name to avoid name inconsistency. No strong opinion, was just a quirk